lib/pharos/phases/configure_client.rb

```diff
@@ -17,28 +14,6 @@ def call
         cluster_context['previous-config'] = build_config(config_map)
       end
 
-      def save_config_locally
```

Comment:
How do we load config for kube client now?

Reply:
I think I had one and it worked because of that.